lux/utils/utils.py

```diff
@@ -92,8 +92,15 @@ def check_if_id_like(df, attribute):
             and str_length_uniformity
         )
     else:
-        # TODO: Could probably add some type of entropy measure (since the binned id fields are usually very even)
-        return high_cardinality and (attribute_contain_id or almost_all_vals_unique)
+        if len(df) >= 2:
+            diff = df[attribute].diff()
+            evenly_spaced = all(diff.loc[1:] == diff.loc[1])
```

Comment:
This line fails on examples with indexes. 
```python
python -m pytest tests/test_action.py
```
![image](https://user-images.githubusercontent.com/5554675/105013651-70358e00-5a7a-11eb-819d-89ff4699ccf8.png)